app/Domain/Integrations/FormRequests/StoreIntegrationUrlRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\FormRequests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+final class StoreIntegrationUrlRequest extends FormRequest
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
```

Comment:
Not sure how stricted these rules should be, but should we also verify if:
- The environment is known?
- The type is known?
- The url is a valid HTTPS url?

Reply:
Yes 👍🏻 For storing and updating aswell

Reply:
`environment` and `type` will fail on the conversion to its enum value:

```php
Environment::from($request->input('environment')),
IntegrationUrlType::from($request->input('type')),
```

The frontend also only let's you choose the supported values.

So in a worst case scenario if someone tries to send their own modified inertia request, it will fail right after the laravel validation at the enum conversion.